src/mouse_events.cpp

```diff
@@ -597,6 +597,48 @@ void mouse_handler::mouse_motion(int x, int y, const bool browse, bool update, m
 	}
 }
 
+// Hook for notifying lua game kernel of mouse button events. We pass button as
+// a serpaate argument than the original SDL event in order to manage touch
+// emulation (i.e., long touch = right click) and such.
```

Comment:
Very minor quibble, but this seems like it should be e.g., not i.e.

Reply:
yup, it should be!